Updated Glossary Page

swati1717 opened the pull request:
Added HotWax Commerce terminology and definitions on Glossary page in User-manual and Store Operations

swati1717 committed:
Update GLOSSARY.md

Added HotWax Commerce terminology and definitions

swati1717 committed:
Update GLOSSARY.md

Added HotWax Commerce terminology and definitions

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
 
-### Safety Stock
+### Digital Good
+Digital gift cards, also known as e-gift cards, are virtual equivalents of physical gift cards. Customers receive them via email or text message along with a unique serial number that they can use to redeem the value stored in the card. When customers order a digital gift card on Shopify, the order is immediately auto-fulfilled, a serial number is assigned, and the value is loaded to activate the ...
```
as mentioned, this is not only limited to gift card, either change the heading from digital goods to digital gift cards.

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
+### Bundle Product
+A Bundle Product refers to a product composed of multiple components, which are imported from an external system. The Import Bundle Product Components job ensures that these components are accurately reflected in the Order Management System (OMS).  
+The lowest common denominator of the Bundle component inventory available by facility determines the bundle product's availability. The computed inventory of a Bundle product at each facility is then summed to produce the final sellable inventory number.  
+To know more about Bundle Products [Click Here](https://docs.hotwax.co/deployment-and-configurations/v/shopify/additional-resources/kitproducts).
```
Add link form new user manuals

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
+Order Fulfillment Status in HotWax Commerce tracks the current stage of an overall order within the order fulfillment process. This status helps in monitoring and managing orders, indicating whether an Order is created, approved, completed, or canceled.
 
 ### Order Merging
+When a customer places multiple orders that can be fulfilled from the same location. Instead of treating them as separate orders, retailers prefer to merge them into a single shipment, ensuring a smoother and more cost-effective delivery process. Hotwax commerce enables order merging so that orders coming from the same location are fulfilled together.
```
give link for blog

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
-
-Orders for temporarily out-of-stock products, named "backorders" as these items were previously in stock and will be available again soon.
+### Pre-Orders
+Pre-orders are simply advance orders taken for future inventory. These products have not yet been released in the market and are still in planning or production, and thus not in stock. A pre-order is an order placed for a product that is not yet available for purchase or delivery. Customers can reserve the item in advance, and the order will be fulfilled once the product becomes available in stoc ...
```
give link to blog or user manual

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
+Pre-orders are simply advance orders taken for future inventory. These products have not yet been released in the market and are still in planning or production, and thus not in stock. A pre-order is an order placed for a product that is not yet available for purchase or delivery. Customers can reserve the item in advance, and the order will be fulfilled once the product becomes available in stoc ...
 
 ### Pre-Order Parking
+Pre-order parking in HotWax Commerce is a holding area where pre-ordered items are kept until their inventory becomes available. Orders in this queue are not yet ready for fulfillment and wait until the specified stock arrives before being moved to the brokering queue for processing and shipping. This ensures efficient and timely handling of pre-orders based on inventory availability.
```
give link to parking page

Amir-HC commented on user-manuals/GLOSSARY.md:
```diff
@@ -4,117 +4,167 @@ description: >-
 
-The current stage or status of an item within the fulfillment process, including options such as Reserved, Picking, Packed, Shipped or Delivered
+### Safety Stock
+Safety stock helps retailers control how much inventory they promise to online channels. Once a store’s inventory count falls below the designated safety stock quantity, the remaining inventory will be reserved for in-store sales. Retailers often face inventory discrepancies, where a system’s inventory does not match the actual inventory in stores. Keeping safety stock also helps retailers mitiga ...
```
give link to safety stock user manual

swati1717 committed:
Merge branch 'hotwax:user-guides-pub' into user-guides-pub

swati1717 committed:
Update GLOSSARY.md

Updated Links for 
Safety Stock
Pre-Order
Pre-Order Parking
Digital gift Card
Excluded Facility

swati1717 committed:
Update GLOSSARY.md

Updated Links for 
Pre Order
Pre Order Parking
Safety Stock
Excluded Facility
Digital Gift Cards

swati1717 committed:
Merge branch 'hotwax:user-guides-pub' into user-guides-pub

swati1717 committed:
Update GLOSSARY.md

Added Glossary

swati1717 committed:
Update GLOSSARY.md

Added HotWax Commerce terminology and definitions on Glossary page.
